docs/user/ppl/functions/relevance.rst

```diff
@@ -98,6 +98,49 @@ Another example to show how to set custom values for the optional parameters::
     +----------------------+--------------------------+
 
 
+
+MATCH_PHRASE
```

Comment:
Should be MATCH_PHRASE_PREFIX. Also description section is incorrect.

Reply:
What do you think of https://github.com/Bit-Quill/opensearch-project-sql/pull/77/commits/451b8292cc935af0397e5d1b843ce9180d89880?